numbat/src/vm.rs

```diff
@@ -743,6 +834,7 @@ impl Vm {
                             Value::Quantity(q) => q.to_string(),
                             Value::Boolean(b) => b.to_string(),
                             Value::String(s) => s,
+                            Value::DateTime(..) => "DateTimeTodo_1".to_string(),
```

Comment:
This should be implemented for use cases like:
```
print("Current time: {now()}")
```

Reply:
Ahh, I totally failed to notice that string interpolation uses this different codepath.  This should now be fixed